Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ForwardDiff"
 uuid = "f6369f11-7733-5829-9624-2563aa707210"
-version = "0.10.7"
+version = "0.11.0"
```

Comment:
```suggestion
version = "0.10.8"
```
?

Reply:
but it had a compatibility change

Reply:
Yes? How is that breaking?

Reply:
Doesn't General require a 0.x.0 bump with compatibility changes?

Reply:
No.

Reply:
Alrighty then.